critiquebrainz/db/review_test.py

```diff
@@ -31,6 +31,7 @@ def test_review_creation(self):
             entity_id="e7aad618-fa86-3983-9e77-405e21796eca",
             entity_type="release_group",
             text="Testing",
+            rating=100,
```

Comment:
OK. You could also add tests to make sure that all kinds of reviews are stored properly.